pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -366,6 +372,7 @@ func (s *DiscoveryServer) configz(w http.ResponseWriter, req *http.Request) {
 // It is mapped to /debug/adsz
 func adsz(w http.ResponseWriter, req *http.Request) {
 	_ = req.ParseForm()
+	w.Header().Add("Content-Type", "application/json")
```

Comment:
I've had good luck recently serializing proto Structs to JSON then printing that: `s, err := (&jsonpb.Marshaler{}).MarshalToString(struct)`

It's human readable without all the struct garbage.

Reply:
E.g. I use that while debugging listeners (which have multiple `proto.Struct`s in them) all the time.

Reply:
Thanks for the tip - will try it (in a separate PR...)